server/tests/article.test.js

```diff
@@ -13,6 +13,8 @@ import articleData from './testData/article.data';
 dotenv.config();
 const { expect } = chai;
 chai.use(chaiHttp);
+chai.use(require('chai-integer'));
```

Comment:
Nice work bro. But since we imported chaiHttp, I think it would be uniform if we imported and then used it the way we used chaiHttp.

Reply:
+1